src/chrome/content/rules/CCC.de.xml

```diff
@@ -1,58 +1,47 @@
 <!--
-	For rules subject to CAcert, see CCC.de-cacert.xml.
-
 	Other CCC rulesets:
+		+ C3VOC.de.xml
+		+ CCCv.de.xml
 
-		- C3VOC.de.xml
-		- CCCv.de.xml
-
-
-	Nonfunctional hosts in *ccc.de:
-
-		- dasalte ¹
-		- halfnarp.events ¹
-		- wahlcomputer ²
-
-	¹ Shows wuppa
-
-	Problematic hosts in *ccc.de:
-
-		- wiki.chaosradio²
-		- ds ¹
-		- dsrad ¹
-		- wuppa ¹
-		- irc.darmstadt
-		- www.duesseldorf ²
-		- frankfurt ² 
-		- ftp.media ² 
-		 -upload.media ² 
-		- gost.hannover ²
-		- www.hannover ²
-		- pads³
-		- ffhh.pads ³
-		- r0ket.badge.events¹
-		- www.pads ³
-
-	¹ Expired
-	² no valid cert
-	³ incomplete chain
-
-	403:
-		-koeln.media
-
-	Timeout:
-		- noc.events
+	Dropped:
+		- noc.events (fixed by this ruleset)
```

Comment:
This seems not to be `fixed by this ruleset`

Reply:
@J0WI Done.